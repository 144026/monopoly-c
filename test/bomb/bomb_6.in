preset user AQJ
preset option mskip on
preset option sell_bomb on
# new tool house location
preset userloc A 26 0
preset userloc Q 25 0
preset userloc J 25 0
preset gift A bomb 3
# 'Bomb' not alllowed at special map pos 28 (type 2).
bomb 2
step 1
n
step 3
step 3
# buy bomb
preset credit A 50
step 1
2
dump