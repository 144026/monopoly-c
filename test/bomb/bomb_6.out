user AQJ
fund A 10000
fund Q 10000
fund J 10000
credit A 0
credit Q 0
credit J 0
userloc A 28 0
userloc J 28 0
userloc Q 28 0
gift A bomb 4
nextuser Q